### PR TITLE
Adding another resource available on youtube

### PR DESCRIPTION
**Pull request by shehroze-1122**
Hi, I added another great resource for developers available on youtube. Kindly review and add. I have added additional details in the .md file as well.

### CODE REVIEW

File: YouTubeChannels.md [hunk 1]
```diff
@@ -65,6 +65,8 @@
 
 * [Kalle Hallden](https://www.youtube.com/channel/UCWr0mx597DnSGLFk1WfvSkQ): Different topics related to programming such as python, java, linux, etc. The channel doesn't feature many tutorials, however Kalle gives ideas, motivation and quality videos.
 
+* [Clément Mihailescu](https://www.youtube.com/channel/UCaO6VoaYJv4kS-TQO_M-N_g): An Ex-Google Software Engineer, an Ex-Facebook Software Engineer, and the CEO and co-founder of AlgoExpert (algoexpert.io), a website that helps Software Engineers prepare for coding interviews.This YouTube channel is the outlet for all his thoughts on tech, entrepreneurship, and random miscellanea!.There are several competitive coding interviews which he conducts. So anyone looking to prepare for coding interviews to get into big tech companies should definitely check out this channel.
```
Comment (CodingSpecies) [inline YouTubeChannels.md:68]: image
@shehroze1  There seems to be a typo, as seen in the above screenshot, where there seems to be a `!.` Please either remove the `.` or the `!`. Thank you.

**Review by CodingSpecies — CHANGES_REQUESTED** [review]

**Comment by shehroze-1122** [comment]
Thank you for pointing it out. Now it is corrected!

**Review by CodingSpecies — APPROVED** [review]
LGTM, thank you!

**Comment by arjunkharbanda** [comment]
The build is failing so I am sorry I have to close this PR please Re-submit if you can

**Pull request closed without merging**